go.yaml

```diff
@@ -12,8 +12,7 @@ package:
       license: BSD-3-Clause
   dependencies:
     runtime:
-      - build-base
-      - bash
+      - go-1.19
 secfixes:
```

Comment:
Question: Do we need to keep `secfixes` and `advisories` for this meta-package? Or is it enough to have them in the `go-1.19` package this points to?